src/res.h

```diff
@@ -114,8 +114,8 @@ class GameDataDBAsyncResult : public game_data {
         std::future<bool> fut;
         mariadb::result_set_ref res;
         r_string statementName; //Is null if this was not a statement from config
-        game_value callback;
-        game_value callbackArgs;
+        std::vector<game_value> callbacks;
```

Comment:
use 
auto_array<std::pair<game_value, game_value>>
for callbacks and args in single array.

Reply:
changed